UQ-eLIPSE/crucible-components
# Story/vethub-20

Submitting selected option

Button has been added to submit an answer.

The initial state is disabled but when you select an option, it enables.

When the user submits the correct answer, it will highlight the option green. If the user selects the incorrect answer, the wrong selected answer will be red and the correct answer will be highlighted in green.

To test:

- Run `yarn dev` and then navigate to https://localhost:5173
- Check if the submit button is disabled
- Click the option and it should remain selected.
- Click submit
- To test with the correct answer (or wrong if you got it right the first time), refresh the page and retry the steps with a different option. 
- Run `yarn test` to run the updated tests.
